### ISSUE TITLE
Convert Missing Table Header Rule to JS

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-checker#891]
Related issue: [https://3.basecamp.com/3579237/buckets/29333825/card_tables/cards/7671789839](https://3.basecamp.com/3579237/buckets/29333825/card_tables/cards/7671789839)
